src/filters.rs

```diff
@@ -0,0 +1,30 @@
+use crate::models::{Label, Ticket};
+
+pub fn filter_by_assigned_user(user_id: Option<i32>, ticket: &Ticket) -> bool {
+    match user_id {
+        Some(user_id) => ticket.assigned_user.unwrap_or(0) == user_id,
+        None => true,
+    }
+}
+
+pub fn filter_by_title(ticket_title: &Option<String>, ticket: &Ticket) -> bool {
+    match ticket_title {
+        Some(ticket_title) => ticket.title.contains(ticket_title),
+        None => true,
+    }
+}
+
+pub fn filter_by_labels(ticket_labels: &Option<Vec<Label>>, ticket: &Ticket) -> bool {
+    match ticket_labels {
+        Some(ticket_labels) => {
+            let mut includes = false;

```

Comment:
Not working as expected. When first label is not found, it's set to false. When the next one is found, it's set to true, returning a false positive